### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by leedm777**
* Add note about patch needed for certified Asterisk
* Cleaned up certificate install to avoid IP fragmentation problems

### CODE REVIEW

**Comment by jznadams** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -53,6 +65,15 @@ The respoke configuration file follows the same rules and similar patterns to
 that of a typical Asterisk configuration file.  See the 'respoke.conf.sample'
 file for more information.
 
+### DTLS Certificate
+
+The usual Asterisk script for generating a certificates (`ast_tls_cert`)
```
Comment (chadxz) [inline README.md]: 'a certificates' ?

**Comment by chadxz** [comment]
LGTM

**Comment by creslin287** [comment]
+1

**Pull request merged**